src/main/config/config_streamer.c

```diff
@@ -31,6 +31,7 @@ extern uint8_t __config_start;   // configured via linker script when building b
 extern uint8_t __config_end;
 #endif
 
+// @todo this is not strictly correct for F4/F7, where sector sizes are variable
```

Comment:
(IMO flash page size in area used for config is important. But this value may be inherited from linker script, so config location and page size is together
Also page boundaries may be used to infer correct sector and int size
)

Reply:
Okay then it's fine